src/PollinationSDK/Model/SimulationInputs.cs

```diff
@@ -20,47 +20,47 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Converters;
 using System.ComponentModel.DataAnnotations;
-
+using OpenAPIDateConverter = PollinationSDK.Client.OpenAPIDateConverter;
 
 namespace PollinationSDK.Model
 {
     /// <summary>
     /// Simulation Arguments
     /// </summary>
     [DataContract]
-    public partial class AppModulesProjectsDtoSimulationArguments :  IEquatable<AppModulesProjectsDtoSimulationArguments>, IValidatableObject
+    public partial class SimulationInputs :  IEquatable<SimulationInputs>, IValidatableObject
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="AppModulesProjectsDtoSimulationArguments" /> class.
+        /// Initializes a new instance of the <see cref="SimulationInputs" /> class.
         /// </summary>
-        /// <param name="parameters">A list of input parameters.</param>
         /// <param name="artifacts">A list of input artifacts.</param>
-        public AppModulesProjectsDtoSimulationArguments
+        /// <param name="parameters">A list of input parameters.</param>
+        public SimulationInputs
         (
            // Required parameters
-           List<ArgumentParameter> parameters= default, List<AppModulesProjectsDtoSimulationArgumentArtifact> artifacts= default// Optional parameters
+           List<SimulationInputArtifact> artifacts= default, List<ArgumentParameter> parameters= default// Optional parameters
```

Comment:
Could the name of types organized with the same naming convention? 
"SimulationInputArtifact" and "ArgumentParameter" 
To
"SimulationInputArtifact" and "SimulationInputArgument"  OR "ArtifactParameter" and "ArgumentParameter"